src/const/external.ts

```diff
@@ -98,3 +98,15 @@ export const CLASS = {
   HIDDEN: "flicking-hidden",
   DEFAULT_VIRTUAL: "flicking-panel"
 };
+
+/**
+ * An object with all possible {@link Flicking#circularFallback circularFallback}s
+ * @ko Flicking의 {@link Flicking#circularFallback circularFallback}에 설정 가능한 값들을 담고 있는 객체
+ * @type {object}
+ * @property {string} DEFAULT "default"
+ * @property {string} BOUND "bound"
+ */
+export const CIRCULAR_FALLBACK = {
+  DEFAULT: "default",
```

Comment:
If circular does not work, it seems that i do not understand the meaning of `default`. How about `linear`? I confirmed that the camera name is LinearCamera.

Reply:
I chose that name because I thought that `linear` is just an internal term for defining the default scrolling type.
But, I think linear is also okay.